src/Illuminate/Auth/CreatesUserProviders.php

```diff
@@ -24,7 +24,7 @@ trait CreatesUserProviders
     public function createUserProvider($provider = null)
     {
         if (is_null($config = $this->getProviderConfiguration($provider))) {
-            return;
```

Comment:
There is no problem because if we use type-strict, we may get some errors but maybe!!